shogun/multiclass/MulticlassTreeGuidedLogisticRegression.h

```diff
@@ -103,7 +103,7 @@ class MulticlassTreeGuidedLogisticRegression : public LinearMulticlassMachine
 protected:
 
 		/** train machine */
-		bool train_machine(std::shared_ptr<Features> data = NULL) override;
+		bool train_machine(const std::shared_ptr<Features> &data, const std::shared_ptr<Labels>& labs) override;
```

Comment:
These are minor, but would be good to stick to the same style 
```suggestion
		bool train_machine(const std::shared_ptr<Features>& data, const std::shared_ptr<Labels>& labs) override;
```